stdlib/public/core/Integers.swift.gyb

```diff
@@ -3161,16 +3163,14 @@ ${assignmentOperatorComment('%', True)}
   @_inlineable // FIXME(sil-serialize-all)
   @_transparent
   public static func %=(_ lhs: inout ${Self}, _ rhs: ${Self}) {
-    // No LLVM primitives for checking overflow of division
-    // operations, so we check manually.
+    // No LLVM primitives for checking overflow of division operations, so we
+    // check manually.
     if _slowPath(rhs == (0 as ${Self})) {
-      _preconditionFailure("Remainder of division by zero")
+      _preconditionFailure("Division by zero")
```

Comment:
I would argue that the old message was a bit more helpful.

Reply:
This was to align with what you get when you write ‘42 % 0’ or ‘42 / 0’—I think it’s important to be clear that the _remainder_ never overflows; but let me see if I can improve on it a little bit.